proposals/generic-attributes.md

```diff
@@ -0,0 +1,66 @@
+# Generic Attributes
+
+* [x] Proposed
+* [x] Prototype: [N/A](#)
+* [x] Implementation: [In Progress](https://github.com/dotnet/roslyn/tree/features/generic-attributes)
+* [ ] Specification: [Not Started](#)
```

Comment:
This section should be in the issue, but not the spec file

Reply:
Removed the section.